src/components/InlineWidget/InlineWidget.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import faker from 'faker'
+ 
+import InlineWidget from './InlineWidget';
+ 
+describe('InlineWidget', () => {
+  it('renders InlineWidget component', () => {
+    const url = faker.internet.url()
+    render(<InlineWidget url={url} />)
+
+    const iframe = screen.getByTestId('calendly-iframe') as HTMLIFrameElement
```

Comment:
Can we update this (as well as the other tests) to use the element tag instead of a test-id? I'd prefer not to add additional attributes to the DOM nodes unless they are necessary for the library to work.

```typescript
const { container } = render(<InlineWidget url={url} />);

const iframe = container.querySelector("iframe");
```

`querySelector` will also correctly infer the type so that we can remove `as HTMLIFrameElement`.

Reply:
Yeah, I was trying to stay true to the best practices of React Testing Library, but I can do the `querySelector` route.